queries.sql

```diff
@@ -98,4 +98,37 @@ SELECT a.name, a.escape_attempts, o.full_name FROM animals AS a INNER JOIN owner
 
 
 -- Who owns the most animals?
-SELECT COUNT(a.name) AS anim_qty, o.full_name FROM animals AS a INNER JOIN owners AS o ON a.owners_id = o.id GROUP BY o.full_name ORDER BY anim_qty DESC LIMIT 1;
\ No newline at end of file
+SELECT COUNT(a.name) AS anim_qty, o.full_name FROM animals AS a INNER JOIN owners AS o ON a.owners_id = o.id GROUP BY o.full_name ORDER BY anim_qty DESC LIMIT 1;
+
+
+
+----------------------Day-4-Project--------------------------------
+
+-- Who was the last animal seen by William Tatcher?
+SELECT a.name AS last_visited_animal FROM visits JOIN vets ON visits.vets_id = vets.id JOIN animals AS a ON visits.animals_id = a.id WHERE vets.name = 'William Tatcher' ORDER BY visit_date DESC LIMIT 1;
+
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(s.name) AS visit_different_animals, vets.name AS doctor_name FROM specializations JOIN vets ON specializations.vets_id = vets.id JOIN species AS s ON specializations.species_id = s.id WHERE vets.name = 'Stephanie Mendez' GROUP BY vets.name;
```

Comment:
- [ ] something seems incorrect in this case looking at the data you put in the table, Stephanie Mendez saw 4 different animals, it seems you're returning how many species she saw

Reply:
@mikethreels Thank you for your review, the code has been corrected accordingly and result is 4 now.